frontend/src/metabase/lib/query.js

```diff
@@ -1,11 +1,11 @@
 import _ from "underscore";
 import Utils from "metabase/lib/utils";
 
-import * as QUERY from "./query/query";
-import * as FieldRef from "./query/field_ref";
+import * as QUERY from "../../metabase-lib/lib/utils/query/query";
```

Comment:
should we use absolute import like in other places?

Reply:
Sure! Thanks for catching this